.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+  "recommendations": [
+    "dbaeumer.vscode-eslint"
```

Comment:
what is this for? 🤔

Reply:
I had a problem using this tool, and in some projects it wasn’t showing up, it was a test but we can remove it